libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m

```diff
@@ -52,10 +52,16 @@ + (NSString *)localizedString:(NSString *)localizationKey
 
 + (UIImage *)imageNamed:(NSString *)name
 {
-    NSBundle *bundle = [NSBundle bundleForClass:[self class]];
     NSAssert(name != nil, @"name must contain a value.");
+    //Get image from main bundle (this is to allow extracting SDK Assets when using template app as we include SDK Assets as part of the tempalte app)
```

Comment:
If we include these assets as part of the pod, the assets get packaged as a bundle versus a .car. So now we need two different ways (in code) of getting the images for the login screen. To avoid this we are including the SDK Assets(gear icon and back arrow icon) as part of the template app.